tools/ratbagd.py

```diff
@@ -303,6 +305,20 @@ def capabilities(self):
         return self._get_dbus_property("Capabilities")
 
     @GObject.property
+    def name(self):
+        """The name of the profile"""
+        return self._get_dbus_property("Name")
+
+    @name.setter
+    def name(self, name):
+        """Set the name of this profile.
+
+        @param name The new name, as string"""
+        if RatbagdProfile.CAP_WRITABLE_NAME in self.capabilities:
+            return self._set_dbus_property("Name", "s", name)
+        return RatbagErrorCode.RATBAG_ERROR_CAPABILITY
```

Comment:
I'd still prefer an exception here, sorry. If we can't make it work then we can go with this approach, but a a setter should always either set what it is or throw an exception.